### PR TITLE
Completed translation of class io.BufferedIOBase of library/io.po

### PR DESCRIPTION
**Pull request by cschan1828**
Completed translation of class `io.BufferedIOBase` of `library/io.po`

### CODE REVIEW

File: library/io.po [hunk 1]
```diff
@@ -955,28 +955,35 @@ msgid ""
...
+"某些緩衝區，如 :class:`BytesIO`，沒有單一原始串流的概念可從此方法回傳。它們會"
+"引發 :exc:`UnsupportedOperation`。"
 
 #: ../../library/io.rst:565
 msgid ""
 "Read and return up to *size* bytes.  If the argument is omitted, ``None``, "
 "or negative, data is read and returned until EOF is reached.  An empty :"
 "class:`bytes` object is returned if the stream is already at EOF."
 msgstr ""
+"讀取並回傳最多 *size* 個位元組。如果引數被省略、為 ``None``、或為負值，將讀取"
```
Comment (mattwang44) [inline library/io.po]:
```suggestion
"讀取並回傳最多 *size* 個位元組。如果引數被省略、為 ``None`` 或為負值，將讀取"
```

File: library/io.po [hunk 2]
```diff
@@ -1036,19 +1060,27 @@ msgid ""
 "implementation, these bytes may be readily written to the underlying stream, "
 "or held in a buffer for performance and latency reasons."
 msgstr ""
+"寫入給定的 :term:`bytes-like object`，*b*，並回傳寫入的位元組數量（總是等於 "
+"*b* 的長度，以位元組計，因為如果寫入失敗將會引發 :exc:`OSError`）。根據實際的"
+"實作，這些位元組可能會立即寫入底層串流，或出於性能和延遲的原故而被留在緩衝區"
```
Comment (mattwang44) [inline library/io.po]:
```suggestion
"實作，這些位元組可能會立即寫入底層串流，或出於性能和延遲的緣故而被留在緩衝區"
```

File: library/io.po (hunk 1 above)
Comment (cschan1828) [reply library/io.po]: Resolved.

File: library/io.po (hunk 2 above)
Comment (cschan1828) [reply library/io.po]: Resolved.

**Review by mattwang44 — APPROVED** [review]

**Pull request merged**